source/TestContext.cpp

```diff
@@ -17,13 +17,34 @@ class Test;
 
 
 // Constructor to be used when running an actual test.
-TestContext::TestContext(const Test *toRun) : testToRun(1, toRun)
+TestContext::TestContext(const Test *toRun) : callstack({{toRun, 0}})
 {
 }
 
 
 
 const Test *TestContext::CurrentTest() const noexcept
 {
-	return testToRun.empty() ? nullptr : testToRun.back();
+	return callstack.empty() ? nullptr : callstack.back().test;
+}
+
+
+
+bool TestContext::ActiveTestStep::operator==(const ActiveTestStep &rhs) const
+{
+	return this->test==rhs.test && this->step==rhs.step;
```

Comment:
```suggestion
	return test == rhs.test && step == rhs.step;
```

I don't believe this is in the style guide but there aren't a lot of `this->` in the codebase, but idk.